packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -112,6 +119,14 @@ export class Combobox
    */
   @Prop({ reflect: true }) clearDisabled = false;
 
+  /**
+   * When `selectionMode` is `"ancestors"` or `"multiple"`, specifies the display of multiple `calcite-combobox-item` selections
+   * - `"all"` (displays all selections with individual `calcite-chip`s)
+   * - `"fit"` (displays individual `calcite-chip`s that scale to the component's size, including a non-closable `calcite-chip`, which provides the number of additional `calcite-combobox-item` selections not visually displayed).
+   * - `"single"` (display one `calcite-chip` with the total number of selections)
+   */
```

Comment:
Two small copyedit suggestions for consistency:

- Only show one dash `-` before the `"all"` reference - adding commas after the other items for consistency 
    - i.e., `....with individual calcite-chips),`
- Add a period at the end of the single parenthesis
    - i.e., `.... with the total number of selections).`